atlasdb-container-test-utils/src/main/java/com/palantir/atlasdb/containers/CassandraCliParser.java

```diff
@@ -31,9 +31,10 @@ public static int parseSystemAuthReplicationFromCqlsh(String output) throws Ille
         try {
             for (String line : output.split("\n")) {
                 if (line.contains("system_auth")) {
-                    Matcher matcher = Pattern.compile("^.*\\{\"replication_factor\":\"(\\d+)\"\\}$").matcher(line);
+                    String replicationRegex = "^.*('|\")replication_factor('|\"):( '|\")(\\d+)('|\")\\}$";
```

Comment:
Can we have separate regex's for different versions? The one before was painful to read, this one is even worse. (So easy to miss the space before the quote, etc)